src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSlotsMercury.java

```diff
@@ -5,102 +5,45 @@
 import com.wikia.webdriver.common.core.annotations.InBrowser;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.Emulator;
-import com.wikia.webdriver.common.dataprovider.mobile.MobileAdsDataProvider;
+import com.wikia.webdriver.common.core.url.Page;
 import com.wikia.webdriver.common.logging.PageObjectLogging;
 import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.PortableInfobox;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.mobile.MobileAdsBaseObject;
 import org.testng.annotations.Test;
 
+@InBrowser(
+  browser = Browser.CHROME,
+  emulator = Emulator.GOOGLE_NEXUS_5
+)
+@Test(groups = "AdsSlotsMercury")
 public class TestAdsSlotsMercury extends MobileTestTemplate {
   private static final String CREATIVE_IMAGE_URL = "googlesyndication.com/simgad/8216620376696319112";
-  private static final String MOBILE_TOP_LEADERBOARD = "MOBILE_TOP_LEADERBOARD";
-  private static final String MOBILE_IN_CONTENT = "MOBILE_IN_CONTENT";
-  private static final String MOBILE_PREFOOTER = "MOBILE_PREFOOTER";
-  private static final String SRC = "mobile";
   private static final String PORTABLE_INFOBOX = ".portable-infobox";
   private static final String ARTICLE_HEADER = ".wiki-page-header";
   private static final String ARTICLE_BODY = ".article-body";
   private static final String ARTICLE_FOOTER = ".article-footer";
 
-  @Test(
-      groups = "AdsSlotsMercury",
-      dataProviderClass = MobileAdsDataProvider.class,
-      dataProvider = "allSlots"
-  )
-  public void adsAllSlotsOnPage(String wikiName,
-                                String article,
-                                String adUnit) {
-
-    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
-    MobileAdsBaseObject ads = new MobileAdsBaseObject(driver, testedPage);
-    ads.verifyGptIframe(adUnit, MOBILE_TOP_LEADERBOARD, SRC);
-    ads.verifyGptIframe(adUnit, MOBILE_IN_CONTENT, SRC);
+  @Test
+  public void adsAllSlotsOnPage() {
+    Page page = new Page("project43", "SyntheticTests/Mercury/Slots/AllSlots");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "project43" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)